CI failing on TestI2OSPTiming

I think we can disable it and open an issue.

```
--- FAIL: TestI2OSPTiming
```

e.g. https://app.circleci.com/jobs/github/mozilla-services/go-cose/43

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -27,10 +27,37 @@ jobs:
           command: |
             source $HOME/.cargo/env
             make ci
+  "golang-1.13":
+    docker:
+      - image: circleci/golang:1.13
+    working_directory: /go/src/go.mozilla.org/cose/
+    environment:
+      - GO111MODULE: "on" # yaml sees an unquoted on value as "true" but go checks for "on"
+    steps:
+      - checkout
+      - run:
+          name: install rust and cargo
+          command: |
+            sudo apt-get install libnss3-dev
+            curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
+            source $HOME/.cargo/env
+            echo $PATH
+            which cargo
+      - run:
+          name: print rust and cargo version information
+          command: |
+            source $HOME/.cargo/env
+            rustc --version
+            cargo --version
+      - run:
+          name: run tests
+          command: |
+            source $HOME/.cargo/env
+            make ci
 
 workflows:
   version: 2
   build:
     jobs:
-      - "golang-1.11"
       - "golang-1.12"
+      - "golang-1.13"

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,33 +1,5 @@
 version: 2
 jobs:
-  "golang-1.11":
-    docker:
-      - image: circleci/golang:1.11
-    working_directory: /go/src/go.mozilla.org/cose/
-    environment:
-      - GO111MODULE: "on" # yaml sees an unquoted on value as "true" but go checks for "on"
-    steps:
-      - checkout
-      - run:
-          name: install rust and cargo
-          command: |
-            sudo apt-get install libnss3-dev
-            curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
-            source $HOME/.cargo/env
-            echo $PATH
-            which cargo
-      - run:
-          name: print rust and cargo version information
-          command: |
-            source $HOME/.cargo/env
-            rustc --version
-            cargo --version
-      - run:
-          name: run tests
-          command: |
-            source $HOME/.cargo/env
-            make ci
-
   "golang-1.12":
     docker:
       - image: circleci/golang:1.12

--- a/core_test.go
+++ b/core_test.go
@@ -266,7 +266,7 @@ func TestI2OSPTiming(t *testing.T) {
 		elapsed_times []time.Duration
 	)
 	if os.Getenv("CI") == "true" {
-		toleranceNS = int64(5000) // i.e. 5 microseconds
+		toleranceNS = int64(50000) // i.e. 50 microseconds
 		fmt.Printf("I2OSPTiming using larger timing diff in CI of %s", time.Duration(toleranceNS))
 	}
 

